go/client/cmd_status_osx.go

```diff
@@ -5,12 +5,22 @@
 
 package client
 
-import "github.com/keybase/client/go/install"
+import (
+	"strings"
+
+	"github.com/keybase/client/go/install"
+)
 
 func (c *CmdStatus) osSpecific(status *fstatus) error {
 	serviceStatus := install.KeybaseServiceStatus(c.G(), "service", 0, c.G().Log)
 	kbfsStatus := install.KeybaseServiceStatus(c.G(), "kbfs", 0, c.G().Log)
 
+	productVersion, buildVersion, err := c.osVersionAndBuild()
+	if err != nil {
+		c.G().Log.Debug("Error determining OS version: %s", err)
```

Comment:
`return fmt.Errorf("Error determining OS version: %s", err)` ?

or if it's not fatal, maybe it can return after logging

you might want to use `Log.Warning` or `Log.Errorf`